src/bindings.rs

```diff
@@ -22,6 +22,7 @@
 
 use libc::{c_uint, c_ulong, c_void};
 pub const RANDOMX_HASH_SIZE: u32 = 32;
+#[allow(dead_code)]
 pub const RANDOMX_DATASET_ITEM_SIZE: u32 = 64;
```

Comment:
```suggestion
```

Reply:
:+1: I was also thinking to remove it entirely.